src/tenant/tests/test_initialization.py

```diff
@@ -4,6 +4,7 @@
 
 from badges.models import BadgeRarity
 from courses.models import MarkRange
+from utilities import MenuItem
```

Comment:
did you run this test?  haha.  `from utilities.models import MenuItem` is prolly what you meant.  Please save me time reviewing and run tests and flake8 locally before pushing =)

Reply:
Sorry, had to rewrite that section, did it too hastily and forwent the necessary procedures. Bad import has been fixed.